app/src/main/java/com/simpragma/recipe/roomDatabase/RoomDatabaseAdapter.java

```diff
@@ -0,0 +1,75 @@
+package com.simpragma.recipe.roomDatabase;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.simpragma.recipe.recipeapp.R;
+import com.squareup.picasso.Picasso;
+
+import java.util.List;
+
+/**
+ * Created by Madvesha on 2/12/18.
+ */
+
+public class RoomDatabaseAdapter extends RecyclerView.Adapter<RoomDatabaseAdapter.ViewHolder> {
```

Comment:
This is not a database adapter. This is a view adapter. Rename accordingly. I don't think you need a new adapter for `Room`. And FYI, Room is not a database by itself.

Reply:
Change it as RoomViewAdapter